nsv13/code/game/general_quarters/dropship.dm

```diff
@@ -6,7 +6,7 @@
 	icon = 'nsv13/icons/turf/dropship.dmi'
 	icon_state = "brown_fr"
 
-/turf/open/indestructible/dropship
+/turf/open/floor/indestructible/dropship
```

Comment:
/turf/open/floor/indestructible does not exist, you would have to define it to make this work

Reply:
You say this like I have any clue on how to do this

Reply:
fixed